package.json

```diff
@@ -169,12 +169,14 @@
     "got": "11.8.3",
     "handlebars": "4.7.7",
     "hasha": "5.2.2",
+    "hcl2-parser": "1.0.3",
     "ignore": "5.2.0",
     "ini": "2.0.0",
     "js-yaml": "4.1.0",
     "json-dup-key-validator": "1.0.3",
     "json-stringify-pretty-compact": "3.0.0",
     "json5": "2.2.0",
+    "lodash": "4.17.21",
```

Comment:
Why do we need this? 😕

Reply:
This is a leftover from my prototyping. I'm going to remove it.